packages/system/dracut/immucore/build.yaml

```diff
@@ -0,0 +1,20 @@
+image: "alpine"
+
+prelude:
+  - apk add git
+  - |
+    PACKAGE_VERSION=${PACKAGE_VERSION%\+*} && \
+    git clone https://github.com/${GITHUB_ORG}/{{ .Values.name }}.git && cd {{.Values.name}} && git checkout v"${PACKAGE_VERSION}"
+
+steps:
+  - mkdir -p /package/usr/lib/dracut/modules.d
+  - mkdir -p /package/etc/dracut.conf.d/
+  - cp {{.Values.name}}/dracut/28immucore /package/usr/lib/dracut/modules.d/28immucore
+  - cp {{.Values.name}}/dracut/dracut.conf /package/etc/dracut.conf.d/10-immucore.conf
```

Comment:
```suggestion
  - cp -r {{.Values.name}}/dracut/dracut.conf /package/etc/dracut.conf.d/10-immucore.conf
```

Reply:
not in this case, we only want to copy the conf